### PR TITLE
Spring cleaning

### PR DESCRIPTION
**Pull request by G-Rath**
Gave the whole thing a bit of an update.

Also removed the need for `node_modules` to be committed by using `unpgk` instead.

I might break that out into another PR 😬 (done via #2)

### CODE REVIEW

**Comment by G-Rath** [comment]
@bcherny any chance of getting this reviewed?

Let me know if there's anything I can do to help make it easier 🙂

File: index.js [hunk 1]
```diff
@@ -1,18 +1,59 @@
...
+}
+
+const options = {
+  // bannerComment: '',
+  declareExternallyReferenced: true,
+  enableConstEnums: true,
+  // style: { /* prettier config */ },
+  unreachableDefinitions: false,
+  strictIndexSignatures: false
+  // $refOptions: {}
```
Comment (bcherny) [inline index.js]: Mind removing the commented-out options if they're unused?

File: index.js [hunk 2]
```diff
@@ -1,18 +1,59 @@
...
+};
+
+// expose options for advance users
```
Comment (bcherny) [inline index.js:21]: How would an advanced user know to use this? Maybe we could `console.info` instructions?

**Review by bcherny — CHANGES_REQUESTED** [review]
Sorry @G-Rath, I missed this PR!

Mind adding a video of your UI changes to this PR's summary? Lots of file changes, so tough to review otherwise.

**Comment by G-Rath** [comment]
@bcherny All good 😄 

I actually setup the github packages for my fork, so you can review the changes in action yourself here: https://g-rath.github.io/json-schema-to-typescript-browser/

(I figure that might be better than a video).

The bulk of the file changes are the removing of `node_modules`, which I broke out into a second PR #2 - probably best to review & merge that one first, to reduce the diff on this one.

**Comment by bcherny** [comment]
Looks great. Thanks for the contribution, @G-Rath!

**Pull request merged**